pkg/controller/pxcbackup/controller.go

```diff
@@ -495,3 +504,26 @@ func setControllerReference(cr *api.PerconaXtraDBClusterBackup, obj metav1.Objec
 	obj.SetOwnerReferences(append(obj.GetOwnerReferences(), ownerRef))
 	return nil
 }
+
+func (r *ReconcilePerconaXtraDBClusterBackup) isAnotherBackupRunning(cluster *api.PerconaXtraDBCluster, cr *api.PerconaXtraDBClusterBackup) (bool, error) {
+	bcpList := api.PerconaXtraDBClusterBackupList{}
+
+	if err := r.client.List(context.TODO(), &bcpList, &client.ListOptions{Namespace: cluster.Namespace}); err != nil {
+		if k8sErrors.IsNotFound(err) {
+			return false, nil
+		}
+		return false, errors.Wrap(err, "failed to get backup list")
+	}
+
+	for _, bcp := range bcpList.Items {
+		if bcp.Spec.PXCCluster != cluster.Name {
+			continue
+		}
+
+		if bcp.Name != cr.Name && (bcp.Status.State == api.BackupRunning || bcp.Status.State == api.BackupStarting) {
```

Comment:
it is better to check only final/finished states
reason: we can add more statuses in the future
```suggestion
		if bcp.Name != cr.Name && bcp.Status.State != api.BackupFailed && bcp.Status.State != api.BackupSucceeded) {
```